modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -168,6 +168,9 @@ export class Wallet implements IWallet {
       'consolidateAddresses',
       'cpfpFeeRate',
       'cpfpTxIds',
+      'comment',
```

Comment:
Do these belong on the build params?

Reply:
moved to within

Reply:
They were part of `selectParams`, so whatever the solution, they need to be included.

Reply:
they are